WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -783,9 +782,6 @@ public boolean onOptionsItemSelected(final MenuItem item) {
         } else if (itemId == R.id.menu_post_settings) {
             InputMethodManager imm = ((InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE));
             imm.hideSoftInputFromWindow(getWindow().getDecorView().getWindowToken(), 0);
-            if (mShowNewEditor || mShowAztecEditor) {
```

Comment:
The intention here was to make sure the post settings screen gets the most recent featured image, in case it has been changed by tapping an image and setting it featured there. This wasn't supported in the legacy editor, which is why it was only necessary if the visual editor or Aztec were enabled.